openmetadata-ui/src/main/resources/ui/src/components/AddService/Steps/ConfigureService.tsx

```diff
@@ -29,6 +29,29 @@ const ConfigureService = ({
 }: ConfigureServiceProps) => {
   const markdownRef = useRef<EditorContentRef>();
 
+  const validationErrorMsg = (): string => {
+    if (showError.name) {
+      return 'Service name is required';
+    }
+    if (showError.duplicateName) {
+      return 'Service name already exist.';
+    }
+    if (showError.delimit) {
+      return 'Name with delimiters are not allowed';
+    }
+    if (showError.nameWithSpace) {
+      return 'Name with space are not allowed';
+    }
+    if (showError.nameLength) {
+      return 'Name size must be between 1 and 128';
+    }
+    if (showError.specialChar) {
+      return 'Special characters are not allowed';
+    }
+
+    return '';
```

Comment:
Should we return some generic error message for the  default case?

Reply:
@vivekratnavel, I think we don't need that because the above one is based on some validation condition.